File: Python/Simple Programs/Mathematical/lcm.py
a=input("Enter Numbers Saparated by Space : ")
a=a.split()
a=[int(i) for i in a]
mul=1

for i in range(0,len(a)):
	mul*=a[i]

for i in range(max(a),mul+1):
	flag=0
	for j in range(0,len(a)):
		if int(i%a[j])==0:
			flag=flag+1
	if flag==len(a):
		lcm=i
		break
print("LCM of ",a,"is ", lcm)

File: C/Recursion/LCM.c
#include <stdio.h>
int num=1;
int lcm_fun(int m,int a[]){
	int flag=0,j;
	for(j=0;j<3;j++){
		if(num%a[j]==0)
			flag++;
	}
	if(flag==3){
		return num;
	}
	else{
		num++;
		return lcm_fun(m,a);
	}
}
int main(){
	int a[3],mul=1,lcm,i;
	printf("Enter Numbers : ");
	for(i=0;i<3;i++)
		scanf("%d",&a[i]);
	for(i=0;i<3;i++)
		mul*=a[i];
	//calling function
	lcm=lcm_fun(mul,a);
	
	printf("LCM is %d",lcm);
	return 0;
}
